contrib/testing/debug-mount-shell.yaml

```diff
@@ -0,0 +1,9 @@
+# Can be used to connect to the Cilium debug shell from a k8s debugging container
+# e.g. for the Cilium Operator
+#
+# kubectl -n kube-system debug cilium-operator-7dcff8d46c-79sp6 -it --image=localhost:5000/cilium/cilium-dev:local --custom=contrib/testing/debug-mount-shell.yaml --profile=general -- bash
```

Comment:
Could we instead of exporting the volume mount etc. just have `cilium-operator shell` command that'd do something like `cilium-dbg/cmd/shell.go`? This is what I'd eventually like to happen for the agent as well (e.g. cilium-dbg binary goes away, you only have `cilium-agent` that has a `shell` sub-command).

Reply:
haven't thought about this. has it's pros and cons - easier to use - but not as easy to pipe into other linux tooling (except one does it on the client)

Reply:
Refactored this so that `cilium-agent` & `cilium-operator` binaries have the `shell` subcommand